UT4MasterServer.sln

```diff
@@ -1,61 +1,90 @@
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Debug|x64.Build.0 = Debug|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Release|Any CPU.Build.0 = Release|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Release|x64.ActiveCfg = Release|Any CPU
-		{1D02F72E-DB23-4139-9C98-BF253F8198AA}.Release|x64.Build.0 = Release|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Debug|x64.Build.0 = Debug|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Release|Any CPU.Build.0 = Release|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Release|x64.ActiveCfg = Release|Any CPU
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}.Release|x64.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{7FF2E309-D898-4E37-ABF1-DE9BC57B304E} = {19DD982A-59E2-4813-BA34-602494B8E815}
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {D9679DFC-4B8F-43E9-B6C1-49B0FECAA7A6}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32825.248
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UT4MasterServer", "UT4MasterServer\UT4MasterServer.csproj", "{D5B4EA3F-2465-4B7E-8D07-400D9FC7F534}"
+EndProject
+Project("{E53339B2-1760-4266-BCC7-CA923CBCF16C}") = "docker-compose", "docker-compose.dcproj", "{1D02F72E-DB23-4139-9C98-BF253F8198AA}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".Tests", ".Tests", "{19DD982A-59E2-4813-BA34-602494B8E815}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "XUnit.Tests", ".Tests\XUnit.Tests\XUnit.Tests.csproj", "{7FF2E309-D898-4E37-ABF1-DE9BC57B304E}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{47853299-8BA6-460B-9C54-91E40D59B8ED}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+	EndProjectSection
+EndProject
+Project("{E24C65DC-7377-472B-9ABA-BC803B73C61A}") = "UT4MasterServer.Web", "UT4MasterServer.Web\", "{351E5A62-5562-4E30-A70C-93D5236F6C7A}"
+	ProjectSection(WebsiteProperties) = preProject
+		TargetFrameworkMoniker = ".NETFramework,Version%3Dv4.8"

```

Comment:
.NET Framework??

Reply:
That was just added by default when I added an existing website to the solution. It doesn't matter because there is no code for it to compile.

Reply:
Even for Docker?

Reply:
There is nothing in the project for it to compile so it won't generate a dll.